features/seaCreatures/seaCreatureSlice.js

```diff
@@ -0,0 +1,26 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  caught: [],
+};
+
+export const seaCreatureSlice = createSlice({
+  name: 'seaCreatures',
+  initialState,
+  reducers: {
+    toggleCaught: (state, action) => {
+      console.log(action.payload);
+      console.log(state.caught);
+      if (state.caught.includes(action.payload)) {
+        state.caught = state.caught.filter(id => action.payload != id);
+      } else {
+        state.caught = [...state.caught, action.payload];
+      }
```

Comment:
Die ganze Funkion des Reducers könnte eigentlich gern in eine eigene Datei und dann langfristig bitte testen.